CMakeLists.txt

```diff
@@ -134,6 +134,10 @@ endif()
 if(NOT CMAKE_CROSSCOMPILING)
     set(TOOLS_FOLDER ${CMAKE_CURRENT_BINARY_DIR})
     add_definitions(-DTARGET_${ARCH})
+    if(TARGET_BUILD_TYPE STREQUAL "Debug" OR
+       TARGET_BUILD_TYPE STREQUAL "RelWithDebInfo")
+        add_definitions(-D_TARGET_DEBUG)
```

Comment:
What's the advantage in using an auxiliary definition, instead of using e.g. DEBUG ?

Reply:
Host tools are a completely separate build. And by default this will now be a release build. So defining _DEBUG when the tools are built as release is just wrong.